frame/assets/src/lib.rs

```diff
@@ -844,6 +953,76 @@ decl_module! {
 				Ok(())
 			})
 		}
+
+		/// Set the metadata for an asset.
+		///
+		/// NOTE: There is no `unset_metadata` call. Simply pass an empty name, symbol,
+		/// and 0 decimals to this function to remove the metadata of an asset and
+		/// return your deposit.
```

Comment:
mmh, do you think it's worth it to make this explicit in the types by e.g. taking `Option<(Vec<u8>, Vec<u8>, u8)>` as argument (or taking an `Option<MetadataStruct>`)?